include/xtensor/xadapt.hpp

```diff
@@ -802,6 +862,54 @@ namespace xt
             l
         );
     }
+
+    /**
+     * @brief xtensor adaptor for a pointer.
+     *
+     * Construct for example with:
+     *
+     * \code{.cpp}
+     * #include <xtensor/xadapt.hpp>
+     *
+     * std::array<size_t, 2> shape = {2, 2};
+     * std::vector<double> data = {1, 2, 3, 4};
+     *
+     * xt::xtensor_pointer<double*, 2> a = xt::adapt(data.data(), 4, xt::no_ownership(), shape);
+     * \endcode
+     *
+     * @ingroup xt_xadapt
+     * @tparam C The closure type of the pointer (e.g. ``double*``).
```

Comment:
Nitpicking: C is not the closure type of the pointer, but type of the pointer. I think we can even sue the value type, no need to repeat the `*` part. And then we could use T instead of C for the type parameter (C sounds more like a container type than a basic type).

Reply:
Good idea! Done.